src/Microsoft.AspNet.Mvc.Common/StringHelper.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc.Common
+{
+    internal static class StringHelper
```

Comment:
suggest leaving this class in Microsoft.AspNet.Mvc.Razor.Host until / unless we need it elsewhere